src/module.c

```diff
@@ -9763,6 +9763,10 @@ ValkeyModuleString *VM_GetModuleUserACLString(ValkeyModuleUser *user) {
  * The returned string must be released with ValkeyModule_FreeString() or by
  * enabling automatic memory management. */
```

Comment:
These comments are important because they are used for the generated API documentation.

```suggestion
 * enabling automatic memory management.
 *
 * If the context is not associated with a client connection, NULL is returned
 * and errno is set to EINVAL. */
```